protocol/x/clob/types/types.go

```diff
@@ -1 +1,63 @@
 package types
+
+import "math/big"
```

Comment:
Didnt put this in shared types. Since clob types and revshare types dont depend on each other - i didnt think this was necessary